### PR TITLE
Feat/breakpoint linenumbers

### PR DESCRIPTION
**Pull request by tarkah**

### CODE REVIEW

File: crates/boulder/src/builder.rs [hunk 1]
```diff
@@ -176,14 +179,24 @@ impl Builder {
...
+                                    )
+                                    .map(|line_num| format!(" at line {line_num}"))
+                                    .unwrap_or_default();
+
                                     println!(
-                                        "\n{} {}",
+                                        "\n{}{} {}",
                                         "Breakpoint".bold(),
+                                        line_num,
                                         if breakpoint.exit {
```
Comment (ermo) [inline crates/boulder/src/builder.rs:195]: Suggestion: Make this more "principle of least astonishment"-y by switching the text to something like:

- (exit) -> (build will exit on shell exit)
- (continue) -> (build will continue on shell exit)

... perhaps?

**Review by ermo — COMMENTED** [review]
Other than the nit, I think I like this.

File: crates/boulder/src/builder.rs [hunk 2]
```diff
@@ -176,14 +179,24 @@ impl Builder {
...
                                             "(exit)".dim()
```
Comment (ermo) [inline crates/boulder/src/builder.rs:196]: Suggestion: Make this more "principle of least astonishment"-y by switching the text to something like:

    (exit) -> (build will exit on shell exit)
    (continue) -> (build will continue on shell exit)

... perhaps?

**Review by ermo — COMMENTED** [review]
Other than the nit, I like this (I might be repeating myself here!)

**Review by ikeycode — APPROVED** [review]

**Pull request merged**